app/javascript/cybersec_cert_tracker/CybersecCertTrackerInit.jsx

```diff
@@ -48,15 +50,23 @@ function CybersecCertTrackerInit({ userData, children }) {
         <NavbarBrand href="/" className="me-auto">
           CyberSec
         </NavbarBrand>
-        <NavbarToggler onClick={toggleNavbar} className="me-2" />
-        <Collapse isOpen={!collapsed} navbar>
-          <Nav navbar>
-            <NavItem>
-              <NavLink onClick={() => logout()} id="logout_button">Logout</NavLink>
-            </NavItem>
-          </Nav>
-        </Collapse>
```

Comment:
The logout button has been removed. This is causing a cucumber test to fail. Can you add the logout button back?

Reply:
Added it back to the sidebar